UnitySDK/Assets/ML-Agents/Scripts/Timer.cs

```diff
@@ -53,13 +53,31 @@ public class TimerNode
         [DataMember(Name="count")]
         int m_NumCalls = 0;
 
+        /// <summary>
+        /// The total recorded ticks for the timer node, plus the currently elapsed ticks
+        /// if the timer is still running (i.e. if m_TickStart is non-zero).
+        /// </summary>
+        public long CurrentTicks
+        {
+            get
+            {
+                long currentTicks = m_TotalTicks;
+                if (m_TickStart != 0)
+                {
+                    currentTicks += (System.DateTime.Now.Ticks - m_TickStart);
```

Comment:
isn't this equivalent to `2 * m_TotalTicks + (System.DateTime.Now.Ticks - m_TotalTicks)`?

Reply:
nvm, misunderstood.